### ISSUE TITLE
Warnings

### ISSUE THREAD

**Issue by mansoornasir** [kushalvyas/Bag-of-Visual-Words-Python#1]
I don't know why but i am getting a lot of deprecated warnings.

**Comment by hy84besu** [comment]
Same here. @mansoornasir have you found the reason?

**Comment by mansoornasir** [comment]
Yeah, the main reason is because the Scalar function is deprecated and apparently there is a new one. I get around it by commenting it in the base directory of Python.

**Comment by hy84besu** [comment]
Thank you :) Ill try it out!
